scripts/gen_regex.py

```diff
@@ -0,0 +1,90 @@
+import argparse
+import unicodedata
+import chardata
+import pathlib
+
+DATA_PATH = pathlib.Path(resource_filename('wordfreq', 'data'))
+
+def _emoji_char_class():
+    """
+    Build a regex for emoji substitution.  First we create a regex character set
```

Comment:
This docstring is still wrong: the final regex in lumi_science matches {emoji}{the same emoji}*, but this just matches single emoji.